static/lang/en.json

```diff
@@ -298,8 +298,8 @@
   "Country.NetworkProperties.InfoBox.Label.Middleboxes.NotFound": "No middleboxes detected",
   "Country.NetworkProperties.Button.ShowMore": "Show more networks",
   "Search.Sidebar.Domain": "Domain",
-  "Search.Sidebar.Domain.Placeholder": "e.g. twitter.com",
-  "Search.Sidebar.Domain.Error": "Please enter a valid domain name, such as twitter.com",
+  "Search.Sidebar.Domain.Placeholder": "e.g. twitter.com or 1.1.1.1",
+  "Search.Sidebar.Domain.Error": "Please enter a valid domain name or IP address, such as twitter.com or 1.1.1.1",
```

Comment:
We should remember to update the master translations file with the updated strings. cc @sarathms

Reply:
@hellais isn't the `static/lang/translations.js` the master translations file? Because I have updated that.

Reply:
We also have an internal spreadsheet which is used to generate all the translation files.

Reply:
okay